utility/README.md

```diff
@@ -1,679 +1,1309 @@
 
-The Access Control List or ACL is the primary mechanism that enables live parameterization of Pocket Network 1.0. Specifically, the ACL maintains the knobs that allow active modification of the behavior of the protocol without any forking. As the name suggests, the ACL also maintains the account(s) permissioned to modify any of the parameters. A value is able to be modified by a ParamChangeMsg from the permissioned owner of that parameter. 
+#### 3.9.1 Parameter Updates
 
+The Access Control List (ACL) is the primary mechanism that enables live parameter configuration. Specifically, the ACL maintains the "feature switches" that allow active modification of the behavior of the protocol without forks. The ACL also maintains the account(s) permissioned to modify said parameters. A value can be modified by a `ParamUpdateMsg` from the permissioned owner of that parameter.
 
-```
-# ParamChange Interface
+```go
 type ParamChangeMsg interface {
-  GetAddress()  Address           # Address of sender account; Must be permissioned on ACL
-  GetParamName() String           # The identifier of the knob
-  GetValue() Big.Int              # The modified value of the parameter
+  GetAddress() Address  # Address of sender & signer; must be permissioned through ACL
+  GetParamName() String # The name of the parameter being updated
+  GetValue() any        # The new value of the parameter being modified
 }
 ```
 
+#### 3.9.2 DAO Treasury
 
-In addition to the ParamChange message, the DAO is able to burn or transfer from a specified DAO Module Account. In practice, this Module Account is used as the on-chain Treasury of the DAO, the limitations and specifics of which are maintained in the DAO 1.0 Constitution document. 
-
+In addition to parameter changes, the DAO is able to burn or transfer from a specified DAO Module Account. In practice, this Module Account is used as the on-chain Treasury of the DAO, the limitations and specifics of which are maintained in the DAO 1.0 Constitution document.
 
-```
-# DAOTreasury Interface
+```go
 type DAOTreasuryMsg interface {
-  GetAddress() Address           # Address of sender account; Must be permissioned on ACL
-  GetToAddress() *Address        # (Optional) receiver of the funds if applicable; nil otherwise
-  Operation() DAOOp              # The identifier of the operation; burn or send
-  GetAmount() Big.Int            # The operation is executed on this amount of tokens
+  GetSrcAddress() Address  # The sender of the funds; must be permissioned through ACL
+  GetDstAddress() *Address # OPTIONAL; the receiver of the funds if applicable
+  GetOperation() DAOOp     # The identifier of the operation; burn or send
+  GetAmount() Big.Int      # The operation is executed on this amount of tokens
```

Comment:
I would consider also an optional short text field used to reference the reason why the action was taken. Could be a URL (IPFS or Forum come to mind)